src/data/equiv/mul_add.lean

```diff
@@ -475,6 +483,10 @@ lemma mul_left_symm_apply (a : G) : ((equiv.mul_left a).symm : G → G) = (*) a
 lemma mul_left_symm (a : G) : (equiv.mul_left a).symm = equiv.mul_left a⁻¹ :=
 ext $ λ x, rfl
 
+@[to_additive]
+lemma _root_.group.mul_left_bijective (a : G) : function.bijective ((*) a) :=
```

Comment:
Ooh, and how about these? Do they need to be in the `group` namespace?

Reply:
I guess they don't  need to be, but I think there is a larger chance of them clashing with other things?